source/extensions/common/wasm/wasm.cc

```diff
@@ -1215,35 +1290,24 @@ void Wasm::registerCallbacks() {
   _REGISTER_PROXY(getSharedData);
   _REGISTER_PROXY(setSharedData);
 
-  _REGISTER_PROXY(getRequestHeader);
-  _REGISTER_PROXY(addRequestHeader);
-  _REGISTER_PROXY(replaceRequestHeader);
-  _REGISTER_PROXY(removeRequestHeader);
-  _REGISTER_PROXY(getRequestHeaderPairs);
-
-  _REGISTER_PROXY(getRequestTrailer);
-  _REGISTER_PROXY(addRequestTrailer);
-  _REGISTER_PROXY(replaceRequestTrailer);
-  _REGISTER_PROXY(removeRequestTrailer);
-  _REGISTER_PROXY(getRequestTrailerPairs);
-
-  _REGISTER_PROXY(getResponseHeader);
-  _REGISTER_PROXY(addResponseHeader);
-  _REGISTER_PROXY(replaceResponseHeader);
-  _REGISTER_PROXY(removeResponseHeader);
-  _REGISTER_PROXY(getResponseHeaderPairs);
-
-  _REGISTER_PROXY(getResponseTrailer);
-  _REGISTER_PROXY(addResponseTrailer);
-  _REGISTER_PROXY(replaceResponseTrailer);
-  _REGISTER_PROXY(removeResponseTrailer);
-  _REGISTER_PROXY(getResponseTrailerPairs);
+  _REGISTER_PROXY(getMapValue);
```

Comment:
It was but I changed it because this data structure (and associated functions) are used for gRPC Metadata as well.  Turns out Envoy Metadata is a different data structure, but Envoy reuses Header for gRPC Metadata, hence the name change.

Reply:
That's because gRPC metadata is transmitted as headers, so it's still a `HeaderMap`, IMHO.

Reply:
Done.